app/org/sagebionetworks/bridge/models/ClientInfo.java

```diff
@@ -134,6 +134,18 @@ public Integer getSdkVersion() {
         return sdkVersion;
     }
     
+    public boolean isSupportedAppVersion(Integer minSupportedVersion) {
+        // If there's no declared client version, it matches anything.
+        if (appVersion != null) {
+            // Otherwise we can't be outside of either range boundary if the boundary
+            // is declared.
+            if (minSupportedVersion != null && appVersion.intValue() < minSupportedVersion.intValue()) {
```

Comment:
Can combine into one statement (appVersion != null && minSupportedVersion != null && ... also if you se the same type as appVersion (Integer), you'll get auto-boxing so it'll just be appVersion < minSupportedVersion